osvr/RenderKit/DirectModeVendors.h

```diff
@@ -238,7 +238,8 @@ namespace renderkit {
             Vendor{"VRG", "VRGate"},
             Vendor{"TSB", "VRGate"},
             Vendor{"VRV", "Vrvana"},
-            Vendor{"TVR", "TotalVision"},
+            Vendor{"TVR", "TotalVision", "Some TotalVision LDT HMD's  with early firmware/EDID data"},
```

Comment:
Looks like an extra space in the comment between HMD's and with.

Reply:
The comment says "Added new TotalVision Vendor ID 'TVL'" but the new vendor ID is TVR. Which is the correct one?

Reply:
http://www.uefi.org/PNP_ID_List?search=TVL indicates that TVL is the one that's actually registered with the UEFI forum. However, note that this existing line was here before - presumably HMDs were shipped with TVR (which actually belongs to "TV Interactive Corporation" since 1996) so they've followed our lead when we switched from an unregistered SEN to the registered SVR.  TVL is listed below.

Have a substantial number of these units entered the field? If not, you can leave this line out/remove it and just manually specify (as a param to enter/leave direct mode and as the "vendor" in the display descriptor) "TVR". That said, I'm not sure 1996's "TV Interactive Corporation" is making HMDs so it may be safe to leave this in.